extensions/amp-soundcloud/1.0/README.md

```diff
@@ -0,0 +1,224 @@
+
+### Web Component
+
+You must include each Bento component's required CSS library to guarantee proper loading and before adding custom styles. Or use the light-weight pre-upgrade styles available inline. See [Layout and style](#layout-and-style).
+
+The examples below demonstrate use of the `<bento-soundcloud>` web component.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-soundcloud
+```
+
+```javascript
+import '@ampproject/bento-soundcloud';
+```
+
+[/example]
+
+#### Example: Include via `<script>`
+
+[example preview="top-frame" playground="false"]
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+  <!-- These styles prevent Cumulative Layout Shift on the unupgraded custom element -->
+  <style data-bento-boilerplate>
+    bento-soundcloud {
+      display: block;
+      overflow: hidden;
+      position: relative;
+    }
+  </style>
+  <script async src="https://cdn.ampproject.org/v0/bento-soundcloud-1.0.js"></script>
+  <style>
+    bento-soundcloud {
+      aspect-ratio: 1;
+    }
+  </style>
+</head>
+<bento-soundcloud
+  id="my-track"
+  data-trackid="243169232"
+  data-visual="true"
+></bento-soundcloud>
+<div class="buttons" style="margin-top: 8px;">
+  <button id="change-track">
+    Change track
+  </button>
+</div>
+
+<script>
+  (async () => {
+    const soundcloud = document.querySelector('#my-track');
+    await customElements.whenDefined('bento-soundcloud');
+
+    // set up button actions
+    document.querySelector('#change-track').onclick = () => {
+      soundcloud.setAttribute('data-trackid', '243169232')
+      soundcloud.setAttribute('data-color', 'ff5500')
+      soundcloud.removeAttribute('data-visual')
+    }
+  })();
+</script>
+```
+
+[/example]
+
+#### Layout and style
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```html
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/bento-soundcloud-1.0.css">
+```
+
+Alternatively, you may also make the light-weight pre-upgrade styles available inline:
+
+```html
+<style data-bento-boilerplate>
+  bento-soundcloud {
+    display: block;
+    overflow: hidden;
+    position: relative;
+  }
+</style>
+```
+
+**Container type**
+
+The `bento-soundcloud` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children (slides) via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties):
+
+```css
+bento-soundcloud {
+  height: 100px;
+  width: 100%;
+}
+```
+
+#### Attributes
+
+<table>
```

Comment:
Sorry just realized that we're using HTML `<table>` - why aren't we using markdown tables instead?

Reply:
This is legacy from an old documentation format I believe. I didn't want to change too much in one PR, but it really does not need to be a table anymore.